ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -210,3 +206,27 @@ def transform_date_for_solr(date):
         log.error("Exception {} occured on date transformation of {}"
                   .format(e, date))
         return None
+
+
+def get_localized_date(value):
+    """Take an isoformat date and return a localized date, e.g.
+    '24. Juni 2020'.
+    """
+    try:
+        dt = isodate.parse_datetime(value)
+        if isinstance(dt, datetime):
+            return localised_nice_date(dt, show_date=True, with_hours=False)
+    except Exception:
+        return None
```

Comment:
should we also return a `""` here as for get_date_picker_format

Reply:
You're right - that's also what the old version of this helper did. Thanks for catching that!